### PR TITLE
Device Manager: Show if sensor data is available

### PR DESCRIPTION
**Pull request by fb**
This adds labels for Temperature and Humidity to the device dialog if a device provides that data.

### CODE REVIEW

File: src/InfoBoxes/Content/Weather.cpp [hunk 1]
```diff
@@ -47,7 +47,7 @@ UpdateInfoBoxHumidity(InfoBoxData &data)
   }
 
   // Set Value
-  data.UnsafeFormatValue( _T("%d"), (int)basic.humidity);
+  data.UnsafeFormatValue( _T("%2.1f"), (float)basic.humidity);
```
Comment (Turbo87) [inline src/InfoBoxes/Content/Weather.cpp]: this seems unrelated to the device manager changes 🤔

File: src/Dialogs/Device/DeviceListDialog.cpp [hunk 2]
```diff
@@ -385,6 +389,12 @@ DeviceListWidget::OnPaintItem(Canvas &canvas, const PixelRect rc, unsigned idx)
     if (flags.traffic)
       buffer.append(_T("; FLARM"));
 
+    if (flags.temperature)
+      buffer.append(_T("; Temperature"));
+
+    if (flags.humidity)
+      buffer.append(_T("; Humidity"));
```
Comment (Turbo87) [inline src/Dialogs/Device/DeviceListDialog.cpp]: I'm not too sure about this. If we would add every datapoint we receive here this will result in a looong string... 🤔

File: src/InfoBoxes/Content/Weather.cpp (hunk 1 above)
Comment (fb) [reply src/InfoBoxes/Content/Weather.cpp]: true

File: src/Dialogs/Device/DeviceListDialog.cpp (hunk 2 above)
Comment (fb) [reply src/Dialogs/Device/DeviceListDialog.cpp]: I actually had that same thought back when I implemented it. I tried `; T` and `; H`, which looked a bit cryptic.

**Comment by fb** [comment]
Okay, it *results* in a long string. If you changed the language to German ("Luftgeschwindigkeit" instead of "Airspeed") or enabled debug mode the string would get too long even on a high DPI screen. Not sure if it's critical, though.

The flags on the device screen are very helpful to quickly find out if everything is working properly (without looking into the raw data stream, which I have to do at the moment), but abbreviations should be sufficient for that.

**Comment by Turbo87** [comment]
can we maybe group it under a single flag called "Atmosphere" or "Environment" or something like that?

**Comment by fb** [comment]
I rebased onto master and appended an entry in NEWS.txt
